definitions/npm/moment_v2.x.x/flow_v0.25.x-/moment_v2.x.x.js

```diff
@@ -245,7 +245,7 @@ declare class moment$Moment {
   toDate(): Date;
   toArray(): Array<number>;
   toJSON(): string;
-  toISOString(): string;
+  toISOString(keepOffset?: boolean): string;
```

Comment:
This was present in v2.3.x types: 

https://github.com/flow-typed/flow-typed/blob/78c72eef4472905c8248ceeb51842dc912c5435a/definitions/npm/moment_v2.3.x/flow_v0.25.x-/moment_v2.3.x.js#L265-L267